diff --git a/node_modules/@atproto/oauth-client-expo/dist/index.js b/node_modules/@atproto/oauth-client-expo/dist/index.js
index 100a970..6536e1f 100644
--- a/node_modules/@atproto/oauth-client-expo/dist/index.js
+++ b/node_modules/@atproto/oauth-client-expo/dist/index.js
@@ -1,4 +1,5 @@
-import './polyfill';
+// Polyfill will be imported in app's index.ts
+// import './polyfill';
 export * from '@atproto/oauth-client';
 export { ExpoOAuthClient } from './expo-oauth-client';
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store-ttl.ts b/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store-ttl.ts
index e416050..13daba6 100644
--- a/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store-ttl.ts
+++ b/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store-ttl.ts
@@ -1,4 +1,4 @@
-import { Configuration, MMKV } from 'react-native-mmkv'
+import { Configuration, MMKV, createMMKV } from 'react-native-mmkv'
 import type { SimpleStore, Value } from '@atproto-labs/simple-store'
 import { MMKVSimpleStore, MMKVSimpleStoreOptions } from './mmkv-simple-store'
 
@@ -30,7 +30,7 @@ export class MMKVSimpleStoreTTL<V extends Value>
   }: MMKVSimpleStoreTTLOptions<V> & Configuration) {
     super({ ...config, encode, decode })
 
-    this.#store = new MMKV({ ...config, id: `${config.id}.exp` })
+    this.#store = createMMKV({ ...config, id: `${config.id}.exp` })
     this.#expiresAt = expiresAt
     if (clearInterval) {
       this.#clearTimer = setInterval(() => this.clearExpired(), clearInterval)
diff --git a/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store.ts b/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store.ts
index 91f2b1f..328d2b1 100644
--- a/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store.ts
+++ b/node_modules/@atproto/oauth-client-expo/src/utils/mmkv-simple-store.ts
@@ -1,4 +1,4 @@
-import { Configuration, MMKV } from 'react-native-mmkv'
+import { Configuration, MMKV, createMMKV } from 'react-native-mmkv'
 import type { SimpleStore, Value } from '@atproto-labs/simple-store'
 
 export type MMKVSimpleStoreOptions<V extends Value> = {
@@ -21,7 +21,7 @@ export class MMKVSimpleStore<V extends Value>
     encode,
     ...config
   }: MMKVSimpleStoreOptions<V> & Configuration) {
-    this.#store = new MMKV(config)
+    this.#store = createMMKV(config)
     this.#decode = decode
     this.#encode = encode
   }
